common/src/main/java/com/hammy275/immersivemc/client/config/screen/ItemGuideCustomizeScreen.java

```diff
@@ -97,18 +97,18 @@ private void renderPreview(PoseStack stack, RGBA color, float heightMult, boolea
 
         MultiBufferSource.BufferSource buffer = Minecraft.getInstance().renderBuffers().bufferSource();
         if (ActiveConfig.placementGuideMode == PlacementGuideMode.CUBE || renderSquare) {
-            stack.translate(0, 64f, 0);
+            stack.translate(0, 64f * size, 0);
```

Comment:
From what I can tell, in comparison with an unmodified branch (1.19.x), the extra furnace height is intended. I think we're good to remove this size multiplication, but let me know if you end up doing a comparison and find out otherwise.

Reply:
But this is for the previews in the item guide options menu.

https://github.com/hammy275/immersive-mc/blob/d0f6ea3efb2bcac11e83e1adab8214a7aeaad42d/common/src/main/java/com/hammy275/immersivemc/client/immersive/AbstractImmersive.java#L375C44-L375C44

You're referring to this, right?

Reply:
My bad there, totally misread that. The size multiplication is good as-is here :)